src/javascript/_common/auth.js

```diff
@@ -78,13 +78,12 @@ export const isOAuth2Enabled = () => {
     return false;
 };
 
-export const getLogoutHandler = onWSLogoutAndRedirect => {
-    const oAuth2Logout = OAuth2Logout(onWSLogoutAndRedirect);
-    return oAuth2Logout;
+export const requestOauth2Logout = onWSLogoutAndRedirect => {
+    OAuth2Logout(onWSLogoutAndRedirect);
 };
 
 export const requestSingleLogout = async (onWSLogoutAndRedirect) => {
-    const _requestSingleLogout = async () => {
+    const requestSingleLogoutImpl = async () => {
```

Comment:
Rename to `requestSingleLogoutImpl` to supress warnings
![Screenshot 2024-12-27 at 12 46 34 PM](https://github.com/user-attachments/assets/7f3e5429-3901-4f10-b75e-d417d7e05d44)